config/settings_schema.json

```diff
@@ -1032,10 +1028,6 @@
   {
     "name": "t:settings_schema.styles.name",
     "settings": [
-      {
```

Comment:
Removing the header because we don't need it anymore.

Reply:
Just remember to remove this translation when we've merged 👍🏻 You could probably do it now, but we can update that later too.

Reply:
I was wondering what was the best way to do it since I am working on 2 separate branches. I am keeping a note to do another pass after on that on another PR next week once everything is merged, thank you!